northstar/src/runtime/island/mod.rs

```diff
@@ -113,7 +113,7 @@ impl Island {
         let (checkpoint_runtime, checkpoint_init) = checkpoints();
         let groups = groups(&manifest);
         let capabilities = capabilities(&manifest);
-        let seccomp = seccomp_filter(&manifest);
+        let seccomp = seccomp_filter(&manifest).expect("Failed to create seccomp filter");
```

Comment:
What would be a potential error here? Is this fine to `expect`?

Reply:
Since `check_platform_requirements()` fails at compile time now, we do not need any error here at all. I removed it.